netdicom/DIMSEprovider.py

```diff
@@ -59,18 +60,29 @@ def Send(self, primitive, id, maxpdulength):
             self.DUL.Send(pp)
         logger.debug('DIMSE message sent')
 
-    def Receive(self, Wait=False, Timeout=None):
+    def Receive(self, Wait=False, Timeout=120):
```

Comment:
I've changed this to 120 and also used this value for the Peek() loop timeout as discussed. I'm happy to leave it as None, but if I do I'd want to put it as a parameter for every SCU method that uses DIMSE.Receive() also such that I can use the timeout.